src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/services/TestCaseDisplayService.kt

```diff
@@ -122,6 +122,22 @@ class TestCaseDisplayService(private val project: Project) {
             .createProjectScopeChooser(
                 "Insert Test Cases into Class"
             )
+
+        // Warning: The following code is extremely cursed.
```

Comment:
Could you perhaps explain me how exactly this is a cursed code?

Reply:
It uses reflection, which should generally be avoided